templates/layout.html

```diff
@@ -6,8 +6,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>{% block title %}{% endblock %}</title>
-    <link rel="stylesheet" type="text/css" href="../static/css/bootstrap.min.css">

```

Comment:
If you check in main.css, it has the respective import. I'd later on automate it all, so they are all combined and minified for faster delivery. Just thinking of a best way to package the whole app.